app/views/bgeigie_imports/show.html.erb

```diff
@@ -34,15 +34,17 @@
               </a>
             </li>
           <%- end -%>
-          <li>
-            <a href="#auto-approve-status" data-toggle="tab">
-              <%- if @bgeigie_import.would_auto_approve == true -%>
-                <%= t('.auto_approve_true') %>
-              <%- else -%>
-                <%= t('.auto_approve_false') %>
-              <%- end -%>
-            </a>
-          </li>
+          <%- if @bgeigie_import.approved? && !@bgeigie_import.would_auto_approve.nil? -%>
```

Comment:
```suggestion
          <%- if @bgeigie_import.processed? && !@bgeigie_import.would_auto_approve.nil? -%>
```

Seems like we want processed here rather than approved. Will mention it to you when you get back.